lib/Elastica/Query.php

```diff
@@ -86,7 +91,7 @@ public static function create($query)
      *
      * @return $this
      */
-    public function setRawQuery(array $query)
+    public function setRawQuery(array $query): self
```

Comment:
Is this needed or some cleanup?

Reply:
Cleanup. I'll remove them later today and re-add them in a new PR as you requested in your comment above.